.gitignore

```diff
@@ -1 +1,4 @@
 .vagrant*
+
+# IDE
+/.idea/
```

Comment:
nit: I'm wondering if we want to just keep IDE based .gitignore elements, localized not within the repo but leave that to the end-user at a global level with something like [global gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)

Totally non-blocking :smile: -but just was curious :smile: 
As I could see it being beneficial if a large group of contributors leverage IDEs like:
- vscode
- intellij
- neovim/vim, etc.

Reply:
OK, that makes sense.